docs/model_developers/handlers.rst

```diff
@@ -233,28 +233,28 @@ method annotated with this annotation will be cached, similar to the way
 `lru_cache <https://docs.python.org/3/library/functools.html#functools.lru_cache>`_ works. The arguments to
 the method will form the cache key, the return value will be cached. When the method is called a
 second time with the same arguments, it will not be executed again, but the cached result is
-returned instead. To exclude specific arguments from the cache key, use the `ignore` parameter.
+returned instead. To exclude specific arguments from the cache key, use the ``ignore`` parameter.
 
-Cache entries will be dropped from the cache when they become stale.
-The for_version and timeout parameters control this behaviour:
+Cache entries will be dropped from the cache when they become stale. Use the following parameters to control when an entry is considered stale:
+  * Setting ``evict_after_creation=True`` will mark entries as stale after a timeout controlled by the ``timeout`` parameter (5000s by default).
+  * Setting ``refresh_after_access=True`` will reset the expiry time of entries to 60s anytime they're used.
```

Comment:
I don't see a reason to hardcode this to 60s. As a default 60s is ok, but we should give more freedom to the user to configure the duration of the timeout like is the case for the `evict_after_creation` timer.

Wouldn't it be a good idea to change the type of these two parameters to integers, so that we can have a different value for each timer?